presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -0,0 +1,31 @@
+==========
+SHOW STATS
+==========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW STATS FOR table
+    SHOW STATS FOR ( SELECT * FROM table [ WHERE condition ] )
+
+Description
+-----------
+
+Returns stats for the named table or for the results of a (limited) query.
+
+Stats are returned for each column, along with a rollup across all columns
+(the column will be ``NULL`` for this rollup).
+
+=====================   ============
+Column                  Description
+=====================   ============
+column_name             The name of the column
+data_size               The total size in bytes of all of the values in the column
+distinct_values_count   The approximate number of distinct values in the column
```

Comment:
> approximate

all values are approximate. Either add "approximate" in description of every value or none of them

Reply:
@findepi I don't think that's true. All of the other values should be deterministic and accurate (at least for any stats produced by Presto on write).